src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -289,8 +297,28 @@ public void paintComponent(Graphics g) {
 			g2.setColor(Color.BLUE);
 			g2.fillRect(0, 0, this.getWidth(), this.getHeight());
 			super.paintComponent(g); // replace current painting
+<<<<<<< Updated upstream
```

Comment:
The stuff with `<<<<<<<` or `=======` or `>>>>>>>` is left over from merge conflicts.  You'll need to fix that before we can merge.  It probably can't even compile the way it is now.